js/blockchaincom.js

```diff
@@ -496,20 +497,30 @@ module.exports = class blockchaincom extends Exchange {
     }
 
     async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetFees ();
         //
         //     {
         //         makerRate: "0.002",
         //         takerRate: "0.004",
         //         volumeInUSD: "0.0"
         //     }
         //
-        await this.loadMarkets ();
-        const response = await this.privateGetFees ();
-        return {
-            'maker': this.safeNumber (response, 'makerRate'),
-            'taker': this.safeNumber (response, 'takerRate'),
+        const makerFee = this.safeNumber (response, 'makerRate');
+        const takerFee = this.safeNumber (response, 'takerRate');
+        const result = {
             'info': response,
         };
+        for (let i = 0; i < this.symbols.length; i++) {
+            const symbol = this.symbols[i];
+            result[symbol] = {
+                'info': {},
```

Comment:
Let's put the `response` here, but not in `result`.

From `fetchTradingFees` we should return this:

```
{
    'BTC/USDT': {
        'info': { ... raw response for this market, or the entire exchange response ... },
        'symbol': 'BTC/USDT',
        'maker': XXX,
        'taker': YYY,
    },
    'ETH/USDT': {
        'info': { ... raw response for this market, or the entire exchange response ... },
        'symbol': 'ETH/USDT',
        'maker': XXX,
        'taker': YYY,
    },
}
```

Not this:

```JavaScript
{
    'info': { ... },
    'BTC/USDT': {
        'info': {},
        'symbol': 'BTC/USDT',
        'maker': XXX,
        'taker': YYY,
    },
    'ETH/USDT': {
        'info': {},
        'symbol': 'ETH/USDT',
        'maker': XXX,
        'taker': YYY,
    },
}
```